### PR TITLE
Prepare for product selection

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

**Review by lslezak — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -4,8 +4,23 @@ skelcd-control-leanos
...
 See also the [documentation for the `control.xml` file][1].
 
 [1]: https://github.com/yast/yast-installation/blob/master/doc/control-file.md
+
+## How It Works
+
+LeanOS integrate multiple previously independ products. LeanOS have its control.xml file
+which have reasonable defaults, except partitioning. And its workflow ends up with product selection client.
+And here comes into play product specific installation.xml. It is common add-on installation xml, that can do
+everythin any add-on can do. Also LeanOS can be installed itself, so it have its control.xml and
```
Comment (teclator) [inline README.md]: `everything`

`so it has its control.xml`

Comment (jreidinger) [reply README.md]: thanks

Comment (teclator) [reply README.md]: In general it should need a whole review of the third person use .. or not used at all :P

Comment (teclator) [reply README.md]: What about something like:

LeanOS adds the capability to install independent products with the same media.

It has its own control.xml file which has reasonable defaults for almost all except partitioning.

Its workflow ends up with a product selection client, and there is where comes to play the `installation.xml` specific product file. It is a common add-on installation xml that can do everything an add-on can do. You can read more about the add-on installation.xml in this [document](https://github.com/yast/yast-installation/blob/master/doc/control-file.md#control-sectionadd-on-product-installation-workflow-specification)

But LeanOS is a product itself so apart of the `control.xml` file it also has its own `installation.xml`.

File: README.md [hunk 2]
```diff
@@ -4,8 +4,27 @@ skelcd-control-leanos
...
+It has its own control.xml file which has reasonable defaults for almost all except partitioning.
+
+Its workflow ends up with a product selection client, and there is where comes to play the
+installation.xml specific product file. It is a common add-on installation xml that can do everything
+an add-on can do. You can read more about the add-on installation.xml in
+[this document](https://github.com/yast/yast-installation/blob/master/doc/control-file.md#control-sectionadd-on-product-installation-workflow-specification).
+
+But LeanOS is a product itself so apart of the control.xml file it also has its own installation.xml.
+
+Product selector in installator basically search for any product. Then for each product it try to
```
Comment (lslezak) [inline README.md]: Sounds too complicated to me, I'd replace the whole paragraph with this text:

The product selector dialog in the installation displays all products for which exists a `system-installation() = <product_name>` provides dependency. When a product is selected then its corresponding package providing the system installation is unpacked and its `installation.xml` file is used to continue with the installation.

**Review by lslezak — CHANGES_REQUESTED** [review]

**Review by lslezak — APPROVED** [review]

**Pull request merged**